src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/Marshaling/JSMarshalerArgument.Int52.cs

```diff
@@ -36,7 +36,7 @@ public void ToJS(long value)
         {
             if (value < I52_MIN_VALUE || value > I52_MAX_VALUE)
             {
-                throw new OverflowException($"Overflow: value ${value} is out of ${I52_MIN_VALUE} ${I52_MAX_VALUE} range");
+                throw new OverflowException(SR.Format(SR.ValueOutOf52BitRange, value, I52_MIN_VALUE, I52_MAX_VALUE));
```

Comment:
Maybe use something like https://github.com/dotnet/runtime/blob/b34029154a1da85e1ce37471bd93eb25b015b0db/src/libraries/System.Runtime.Numerics/src/Resources/Strings.resx#L108-L110

Reply:
It's not 64bits range it's 52bits range. There is no dotnet data type for that.